graph-manager-docs/source/slack-integration.md

```diff
@@ -3,68 +3,75 @@ title: Integrating Graph Manager with Slack
+Apollo Graph Manager can send notifications about your data graph to your team's Slack workspace. This enables you to act on any potential issues as soon as they arise.
 
-1. Send a [**daily snapshot**](#daily-reports) of the request rate, error rate, and performance latency of your graph.
-2. Send [**notifications**](#notifications) that are triggered on thresholds like error percentage and performance latency.
-3. Send [**schema notifications**](#schema-notifications) whenever an update lands on your schema.
+These notifications include:
 
-## Configure the integration
+* [Daily reports](#daily-reports) of your graph's request rate, error rate, and latency
+* [Schema change notifications](#schema-change-notifications) whenever your graph's schema is updated
+* [Performance alerts](#performance-alerts) whenever a metric such as error percentage or request latency exceeds a particular threshold (this feature requires a [paid plan](https://www.apollographql.com/pricing/))
 
-The Apollo Slack integration is set up and configured through the Graph Manager UI. If you do not yet have account, [**follow this guide**](https://www.apollographql.com/docs/apollo-server/features/metrics/#Apollo-Graph-Manager) to get started connecting your server to Graph Manager.
+## Setup
 
-If you already have a Graph Manager account, [**log in**](https://engine.apollographql.com) and ––
+> If you don't have a Graph Manager account yet, [get started](getting-started/).
 
-1. Select the service you want to turn on Slack notifications for.
-1. Visit the "Integrations" tab in the left nav.
-1. You'll notice a "Reporting Channels" section at the bottom of this page. Click the "Add channel" button and follow the steps in the Graph Manager UI to get a webhook from Slack.
+### Connecting a Slack channel
 
-Once you've configured your Slack channel you'll be able to turn on daily reports snapshotting and configure notifications in the "General" and "Performance Alerts" sections.
+You configure the Slack integration from the [Graph Manager UI](https://engine.apollographql.com):
 
-![The Integrations tab in Graph Manager](./img/integrations/integrations-tab.png)
+1. Select the graph you want to configure Slack notifications for.
+2. Open the graph's Integrations page in the left nav.
+3. In the Reporting Channels section at the bottom of the Integrations page, click **Configure Slack Channel**. The following dialog appears:
 
-## Daily reports
+    <img class="screenshot" src="./img/integrations/configure-slack-integration.jpg" alt="Modal to set up Slack integration"></img>
 
-Daily reports from Graph Manager are sent out around 9am in whichever timezone you configure them to be in. You turn them on in the "Integrations" tab as shown above. The reports have a set format that gives a birds-eye view of what your GraphQL API delivered in the previous day:
+4. Specify the name of the Slack channel you want to push notifications to. 
```

Comment:
Should we recommend starting the name with a `#` or too minute a detail?

Reply:
Hmm. It looks like the name of this can be _anything_. It is entirely the webhook URL that determines the channel that's posted to. I'll updated this language.